patches/server/0925-Add-Offline-PDC-API.patch

```diff
@@ -0,0 +1,99 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Lulu13022002 <41980282+Lulu13022002@users.noreply.github.com>
+Date: Sat, 9 Jul 2022 17:28:42 +0200
+Subject: [PATCH] Add Offline PDC API
+
+
+diff --git a/src/main/java/net/minecraft/world/level/storage/PlayerDataStorage.java b/src/main/java/net/minecraft/world/level/storage/PlayerDataStorage.java
+index 601f8099f74e81c17600566b3c9b7a6dd39c9bcb..bf7c29c481539d796521de725e17a58955410866 100644
+--- a/src/main/java/net/minecraft/world/level/storage/PlayerDataStorage.java
++++ b/src/main/java/net/minecraft/world/level/storage/PlayerDataStorage.java
+@@ -47,6 +47,22 @@ public class PlayerDataStorage {
+         }
+ 
+     }
++    // Paper start
++    public void save(java.util.UUID uuid, CompoundTag tag) {
```

Comment:
Unsure how to properly do this, this method is pretty much just a copy of the existing `PlayerDataStorage#save(Player)`.
Could reduce duplicate code here.